docker-compose.yml

```diff
@@ -13,11 +13,10 @@ services:
       - MYSQL_RANDOM_ROOT_PASSWORD=yes
 
   selenium:
-    image: selenium/standalone-firefox-debug:3.141.59-zirconium
-    volumes:
-      - /dev/shm:/dev/shm
+    image: selenium/standalone-firefox:4.25
     ports:
-      - "5900:5900"
+      - "4444:4444"  # Selenium WebDriver port
+      - "5900:5900"  # VNC Viewer port (optional for VNC)
```

Comment:
I don't think `ports: ...` is needed in the Docker compose file since the containers only communicate between themselves. Can you test the PR without these three lines ?